src/mbgl/gl/context.cpp

```diff
@@ -457,10 +457,16 @@ Framebuffer Context::createFramebuffer(const Texture& color) {
 Framebuffer
 Context::createFramebuffer(const Texture& color,
                            const Renderbuffer<RenderbufferType::DepthComponent>& depthTarget) {
+    if (color.size != depthTarget.size) {
+        throw std::runtime_error("Renderbuffer size mismatch");
+    }
     auto fbo = createFramebuffer();
     bindFramebuffer = fbo;
+    // TODO: revert this; just for debugging on CI:
+    GLuint depthInt = depthTarget.renderbuffer;
+    Log::Info(Event::General, "bound framebuffer: %d; renderbuffer name: %d", bindFramebuffer, depthInt);
```

Comment:
Got this on Qt app's stdout :warning:

Reply:
oops 🙈 good catch, thanks @brunoabinader!